DuckDuckGo/SettingsViewController.swift

```diff
@@ -117,18 +115,14 @@ class SettingsViewController: UITableViewController {
     private var shouldShowSyncCell: Bool {
         return featureFlagger.isFeatureOn(.sync)
     }
-    
-    private lazy var shouldShowAppTPCell: Bool = {
-#if APP_TRACKING_PROTECTION
-        return featureFlagger.isFeatureOn(.appTrackingProtection)
-#else
-        return false
-#endif
-    }()
```

Comment:
Interesting, is this being removed permanently here? Note that AppTP may be restored later so I'm hesitant to remove pieces like this.